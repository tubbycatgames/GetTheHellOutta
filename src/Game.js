import React from "react";
import styled from "styled-components/native";
import { GameEngine } from "react-native-game-engine";
import { StatusBar } from "react-native";

import systems from "./systems";

const FullScreenEngine = styled(GameEngine)`
  flex: 1;
  background-color: #000;
`;

const Game = () => (
  <FullScreenEngine systems={systems} entities={{}}>
    <StatusBar hidden={true} />
  </FullScreenEngine>
);

export default Game;
