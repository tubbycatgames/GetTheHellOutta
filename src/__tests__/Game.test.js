import React from "react";
import renderer from "react-test-renderer";

import Game from "../Game";

describe("Game", () => {
  it("should render the full game", () => {
    const game = renderer.create(<Game />);
    expect(game.toJSON()).toMatchSnapshot();
  });
});
