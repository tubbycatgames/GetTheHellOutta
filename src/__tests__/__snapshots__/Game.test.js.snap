// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game should render the full game 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      Array [
        Object {
          "backgroundColor": "#000",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ],
    ]
  }
>
  <View
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  />
  <View
    pointerEvents="box-none"
    style={
      Array [
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
        },
        Object {
          "height": 1334,
          "width": 750,
        },
      ]
    }
  />
</View>
`;
